packages/create-bison-app/template/package.json.ejs

```diff
@@ -32,7 +32,8 @@
     "g:test:factory": "hygen test factory --name",
     "g:test:request": "hygen test request --name",
     "g:test:util": "hygen test util --name",
-    "lint": "yarn eslint . --ext .ts,.tsx --fix --ignore-pattern tmp",
+    "lint": "yarn eslint . --ext .ts,.tsx --ignore-pattern tmp",
+    "lint:fix": "yarn eslint . --ext .ts,.tsx --fix --ignore-pattern tmp",
```

Comment:
I prefer this separation! Great add to not just pull out the ability to fix, but to create another distinct script.

Reply:
If you want to make sure both scripts do the same thing, except for fixing lint errors, you could do:

```
lint:fix": "yarn lint --fix"
```